### DISCUSSION TITLE
v0.45.30-beta 不使用meta内核时自动跳过一些不支持的自定义设置

### DISCUSSION THREAD

**Discussion by ForestL18** [vernesong/OpenClash#2481] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/45709305/173790173-fa66485d-8a33-4f84-b746-4745b7736d83.png)

这个意思是指以后只有meta内核能自定义DNS了吗

**Comment by ForestL18** [comment]
@vernesong

**Comment by kingsey** [comment, marked as answer]
指定DNS的策略组和自定义DNS，是两个意思
其他内核本来就没有指定DNS策略组的功能，只是适配Meta之后多了个提示罢了，不用Meta就和以前是一样的。

**Reply by ForestL18** [reply]
名字起的过于相似，meta内核我还不是很了解，所以才有此问

**Reply by shijiezhou1** [reply]
所以解决了吗 这个提示有什么风险
